tests/unit/s2n_server_hello_retry_test.c

```diff
@@ -91,32 +84,26 @@ int main(int argc, char **argv)
             + COMPRESSION_METHOD_SIZE;
 
         server_conn->actual_protocol_version = S2N_TLS13;
+        server_conn->server_protocol_version = S2N_TLS13;
+        server_conn->client_protocol_version = S2N_TLS13;
         server_conn->secure.cipher_suite = &s2n_ecdhe_ecdsa_with_aes_128_gcm_sha256;
         server_conn->secure.server_ecc_evp_params.negotiated_curve = s2n_all_supported_curves_list[0];
         server_conn->secure.client_ecc_evp_params[0].negotiated_curve = s2n_all_supported_curves_list[0];
         EXPECT_SUCCESS(s2n_ecc_evp_generate_ephemeral_key(&server_conn->secure.client_ecc_evp_params[0]));
 
-        s2n_set_connection_hello_retry_flags(server_conn);
+        EXPECT_SUCCESS(s2n_set_connection_hello_retry_flags(server_conn, 1));
```

Comment:
Can you define the magic number 1 here?

Reply:
`1` would be `HELLO_RETRY_MSG`

Reply:
So could you make a constant or #define?